apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1240,6 +1241,27 @@ API.v1.addRoute(
 	},
 );
 
+API.v1.addRoute(
+	'users.isMuted',
+	{ authRequired: true },
+	{
+		async get() {
+			const user = await getUserFromParams(this.queryParams);
+			const room = await Rooms.findOne(this.queryParams.roomId);
+
+			if (!room) {
+				throw new Error('invalid-room');
+			}
```

Comment:
Could you please use `ajv` to validate this endpoints' params instead?
(Check `isUsersInfoParamsGetProps` for reference)

Reply:
Done!